web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v370/migrate-default.sql

```diff
@@ -4,3 +4,12 @@ DELETE FROM Settings WHERE name = 'ui/config';
 
 UPDATE Settings SET value='3.7.0' WHERE name='system/platform/version';
 UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
+
+
```

Comment:
You should also drop status you're removing no ?
Cf. https://github.com/geonetwork/core-geonetwork/pull/3600/files#diff-16e255abcd1f1a2212754b261c3bf01fL526